pwndbg/wrappers/__init__.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import pwndbg.which
+
+readelf_path  = pwndbg.which.which("readelf")
+checksec_path = pwndbg.which.which("checksec")
+file_path     = pwndbg.which.which("file")
+
+
+def OnlyWithFile(cmd):
```

Comment:
Pick a different name (perhaps `OnlyWithCommand`), and use a class-style decorator.  This triple-nesting is pretty hacky, and doesn't make correct use of `functools.wraps`.